### DISCUSSION TITLE
Code example

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [ORelio/Minecraft-Console-Client#2051] [Q&A]
Hey there, 
I have found multiple discussions about left/right clicking on a item in the hub, that it will open an inventory and you should click on the right item to go to the right server. The problem is, that I cant get it working. I see the next possible options todo this:

/useitem
/inventory container list
/inventory container click <slot>
/inventory <inventoryID> click <slot-you-want-to-click> <left|right>

there is already an example given.
/inventory 0 click 36 left
this will click in the playerinventory on slot 36.
but I don't know what the id is of the server selector.

So, what I want to know:
1: How can I right/left click on a item in slot 4?
2: How can I click on an item in the server selector in slot 13?

~Jochem

**Comment by AdiletTermux1011** [comment]
/changeslot 4
/useitem
/inventory 1 click 13 left

Try this, if it doesn't work write back
![Screenshot_5](https://user-images.githubusercontent.com/53665526/174293497-998e9c46-92ba-4b81-a156-f242834975cb.png)

**Reply by DevBlokJochem** [reply]
![image](https://user-images.githubusercontent.com/71212639/174341374-36933ce5-a6c7-4c3d-8aa7-e58a54f3c1be.png)
![image](https://user-images.githubusercontent.com/71212639/174341449-3ff90478-7205-46ba-95df-945d21c45587.png)

This is the setting me nu. I see that my account has the netherstar in his hand, so the changeslot works. But he isnt send to the right server. 
The inventory slot is 13 right?

**Reply by AdiletTermux1011** [reply]
Can you send a screenshot of the console here?

**Reply by DevBlokJochem** [reply]
There is nothing in the console. Only the join message. But nothing after that. If you want, I can send a screenshot when im behind my pc.

**Reply by DevBlokJochem** [reply]
![image](https://user-images.githubusercontent.com/71212639/174443028-167129b3-4e3c-44b9-a243-00b38ad48f9d.png)
